demo/emails/airbnb-review.tsx

```diff
@@ -13,7 +13,7 @@ import {
   Text,
 } from '@react-email/components';
 import * as React from 'react';
-
+//just want to let you know, I am working on it and when I am done you'll be satisfied.
```

Comment:
Hi,
This does not seem to be useful in the context of this project.

Reply:
Thanks for flagging it @antoineharel!